schemas/2019-04-01/deploymentTemplate.json

```diff
@@ -1100,6 +1100,9 @@
                 ]
               }
             ]
+          },
+          { 
+            "$ref": "https://schema.management.azure.com/schemas/2019-04-01/generatedResources.json"
```

Comment:
>"$ref": "https://schema.management.azure.com/schemas/2019-04-01/generatedResources.json" [](start = 12, length = 88)

looks like this causes a test failure.

Reply:
Seems like it was unhappy with an empty "oneOf" statement. I've used `npm run generate-all` to add the previously auto-generated resources to this file.